blocks/responsive-split-layout/responsive-split-layout.css

```diff
@@ -1,29 +1,93 @@
 .responsive-split-layout {
   --split-layout-spacing: 1.5rem;
   --split-layout-content-spacing: 0.5rem;
+  --split-layout-column-gap: 1.25rem;
 
-  display: flex;
-  flex-direction: column;
-  gap: 1.25rem;
-  margin-block-end: var(--split-layout-spacing);
+  width: 100vw;
+  margin-inline: calc(-1 * var(--page-gutters));
 
   @media (min-width: 48rem) {
-    flex-direction: row;
-    gap: 2rem;
+    --split-layout-column-gap: 2rem;
     --split-layout-spacing: 2rem;
-    margin-block-start: var(--split-layout-spacing);
-    margin-block-end: calc(var(--split-layout-spacing) * 2);
   }
 
   @media (min-width: 80rem) {
-    gap: 3rem;
+    --split-layout-column-gap: 3rem;
     --split-layout-spacing: 4rem;
     --split-layout-content-spacing: 1.0rem;
   }
 
   @media (min-width: 105rem) {
-    gap: 12.625rem;
     --split-layout-spacing: 5rem;
+    --split-layout-column-gap: 12.625rem;
+    margin-inline: calc(-0.5 * calc(100vw - var(--page-max-width)));
+  }
+
+  /* 
+    Optional gradient background variant (Who We Are page).
+    - Gradient color applied to custom property via content and JS.
+    - Additional padding around contents, wider image, and less column gap.
+  */
+  &.responsive-split-layout--gradient {
+    --split-layout-spacing: 2.5rem;
+    --split-layout-gradient-color: var(--split-layout-gradient-color-light, transparent);
+
+    body.color-scheme-dark & {
+      --split-layout-gradient-color: var(--split-layout-gradient-color-dark, transparent);
+    }
+
+    @media (min-width: 48rem) {
+      --split-layout-column-gap: 2rem;
+      --split-layout-spacing: 3.25rem;
+    }
+
+    @media (min-width: 80rem) {
+      --split-layout-spacing: 4rem;
+    }
+
+    @media (min-width: 105rem) {
```

Comment:
is it possible to put these at either the top or bottom of the block so it's easier to tell that they are on the`--gradient` class and not the `color-scheme-dark` class?

Reply:
I've moved them up. I previous had this with all the custom property definitions first.